tests/queries/0_stateless/02273_full_sort_join.sql.j2

```diff
@@ -38,7 +36,7 @@ SET max_bytes_in_join = '{% if join_algorithm == 'grace_hash' %}10K{% else %}0{%
 SELECT '-- {{ join_algorithm }} --';
 SET join_algorithm = '{{ join_algorithm }}';
 
-{% for block_size in range(1, table_size + 1) -%}
+{% for block_size in range(1, table_size + 1, 4) -%}
```

Comment:
refer to https://github.com/ClickHouse/ClickHouse/commit/50d27cc218652c71e648bf5b97c9bb24ca170f06.

The test will run timeout with the default step size, since more sqls will run.